homeassistant/components/advantage_air/climate.py

```diff
@@ -189,7 +175,11 @@ async def async_set_hvac_mode(self, hvac_mode: HVACMode) -> None:
 
     async def async_set_fan_mode(self, fan_mode: str) -> None:
         """Set the Fan Mode."""
-        await self.async_update_ac({"fan": HASS_FAN_MODES.get(fan_mode)})
+        if fan_mode == FAN_AUTO and self._ac.get(ADVANTAGE_AIR_AUTOFAN_ENABLED):
```

Comment:
What happens if fan_mode is auto but autofan has been disabled.
Do we need another elif to raise an error then?

Reply:
This PR allows the auto fan more regardless of the MyFan feature, because I've had an issue raised proving you can be in auto mode without the proprietary MyFan mode.